source.go

```diff
@@ -105,9 +107,19 @@ type Source struct {
 	iterator common.Iterator
 }
 
+var (
+	defaultBatchDelay = time.Second * 5
+	defaultBatchSize  = 100000
+)
+
 func NewSource() sdk.Source {
 	// Create Source and wrap it in the default middleware.
-	return sdk.SourceWithMiddleware(&Source{})
+	return sdk.SourceWithMiddleware(&Source{
+		SourceWithBatch: sdk.SourceWithBatch{
+			BatchSize:  &defaultBatchSize,
+			BatchDelay: &defaultBatchDelay,
+		},
```

Comment:
Why doesn't this trigger a change in `connector.yaml` and in the readme?! 😖 This should work, wtf...

Reply:
Oh, because we're using it wrong. We need to change it in the `config` field, that's where all config parameters live. See how it's done in the [kafka connector](https://github.com/ConduitIO/conduit-connector-kafka/blob/8f18cf1875ccf0177a96ca24ed5f7c7cd750552d/source.go#L40-L50) for instance.

Reply:
done